spotify_project/users/models.py

```diff
@@ -0,0 +1,6 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class CustomUser(AbstractUser):
+    spotify_id = models.CharField(max_length=22, primary_key=True)
```

Comment:
You need to run `./manage.py makemigrations`. If that doesn't do anything, then you need to run `./manage.py makemigrations users`. This will create the migration for this user.

In order to actually use this, you may need to delete your current DB and start it over. Django doesn't like switching user models mid project.

Reply:
Thank you for that. I was running `makemigrations` over and over (among other things) and trying to figure out why it wouldn't accept `CustomUser`.